packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -193,7 +193,6 @@ MeetingUtil.isGuestUnjoined = (locusInfo) =>
 
 MeetingUtil.joinMeetingOptions = (meeting, options = {}) => {
   meeting.resourceId = meeting.resourceId || options.resourceId;
-  meeting.setCorrelationId(meeting.id);
```

Comment:
any idea why we are removing this ?

Reply:
@arun3528 correlation id is already set to the meeting id at time of creation of the meeting object: https://github.com/adamweeks/spark-js-sdk/blob/db1acd57384aa6493fb531143db0df14c1572ba7/packages/node_modules/@webex/plugin-meetings/src/meeting/index.js#L357